.rubocop.yml

```diff
@@ -751,3 +750,6 @@ Style/PreferredHashMethods:
 Style/LambdaCall:
   # jbuilder uses this a lot so we want it.
   Enabled: false
+
+RSpec/MultipleMemoizedHelpers:
+  Max: 15
```

Comment:
The default is 10 but I had many problems with this because 10 is a very low number. This time I had 11 memoized helpers on `insert_refunds_spec` and I really couldn't remove any of them. I could just skip it this time, too.

Reply:
Seems fine with me.